### PR TITLE
params: set TTD for mainnet

### PR DESCRIPTION
**Pull request by sandakersmann**
In accordance with: https://github.com/ethereum/execution-specs/pull/585/files

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -57,24 +57,25 @@ var CheckpointOracles = map[common.Hash]*CheckpointOracleConfig{
...
+		ByzantiumBlock:          big.NewInt(4_370_000),
+		ConstantinopleBlock:     big.NewInt(7_280_000),
+		PetersburgBlock:         big.NewInt(7_280_000),
+		IstanbulBlock:           big.NewInt(9_069_000),
+		MuirGlacierBlock:        big.NewInt(9_200_000),
+		BerlinBlock:             big.NewInt(12_244_000),
+		LondonBlock:             big.NewInt(12_965_000),
+		ArrowGlacierBlock:       big.NewInt(13_773_000),
+		GrayGlacierBlock:        big.NewInt(15_050_000),
+		TerminalTotalDifficulty: big.NewInt(58_750_000_000_000_000_000_000),
```
Comment (Uri-Margalit-bloXroute) [inline params/config.go:77]:
```suggestion
		TerminalTotalDifficulty: big.NewInt(0).Mul(big.NewInt(58_750_000_000_000), big.NewInt(1_000_000_000)),
```

**Review by Uri-Margalit-bloXroute — CHANGES_REQUESTED** [review]
the initial value is larger than uint64, you need to use the Mul function to set higher value

**Comment by holiman** [comment]
Closing this in favour of #25528

**Comment by MariusVanDerWijden** [comment]
Hey @sandakersmann,
thank you for your contribution! Sorry that I made a second PR instead of merging yours. 
I've been working on the merge for very long now and making the PR that finally enabled all of this hard work felt right. 
I'm sorry, we're usually prioritizing outside contribution over our own. I hope you understand and that this will not discourage you from contributing.

**Comment by sandakersmann** [comment]
No problem :)

**Pull request closed without merging**